server.js

```diff
@@ -32,6 +32,14 @@ http.createServer( function( request, response ) {
 		return response.end()
 	}
 
+	// robots.txt
+	if ( params.pathname === '/robots.txt' ) {
+		response.writeHead( 200 )
+		response.type('text/plain');
+		response.write( 'User-agent: *' + '\n' + 'Allow: /' )
```

Comment:
This will output a literal `\n`, should be enclosed in double-quotes to get a new line.